src/rotation.rs

```diff
@@ -0,0 +1,59 @@
+use vector::Vector3;
+
+
+/// Euler rotation angles in 3D space.
```

Comment:
Good point! It almost sounds like we should have different types for `TaitBryanAngles` and `ProperEulerAngles`.